### PR TITLE
Added Mullvad

### PR DESCRIPTION
**Pull request by sunknudsen**

### CODE REVIEW

File: profiles/mullvad-adblock-doh.mobileconfig [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+			<key>ProhibitDisablement</key>
+			<false/>
+		</dict>
+	</array>
+	<key>PayloadDescription</key>
+	<string>Configures device to use Mullvad DNS over HTTPS</string>
+	<key>PayloadDisplayName</key>
+	<string>Mullvad DNS over HTTPS</string>
+	<key>PayloadIdentifier</key>
+	<string>com.sunknudsen.mullvad-dns-over-https</string>
```
Comment (paulmillr) [inline profiles/mullvad-adblock-doh.mobileconfig:41]: can you change this to whatever every other mobileconfig is using? makes it easier to to diffs, audits, etc

File: README.md [hunk 2]
```diff
@@ -38,6 +38,8 @@ If you need even more privacy, check out [encrypted-dns over TOR](https://github
...
 | Google                    | 🇺🇸      | No          | [Operated](https://developers.google.com/speed/public-dns/docs/secure-transports) by Google                                             | [HTTPS](https://github.com/paulmillr/encrypted-dns/raw/master/profiles/google-https.mobileconfig),  [TLS](https://github.com/paulmillr/encrypted-dns/raw/master/profiles/google-tls.mobileconfig)                                    |
+| Mullvad                   | 🇸🇪      | Yes          | [Operated](https://mullvad.net/en/help/dns-over-https-and-dns-over-tls/) by Mullvad VPN AB                                             | [HTTPS](https://github.com/paulmillr/encrypted-dns/raw/master/profiles/mullvad-doh.mobileconfig.mobileconfig)                                    |
```
Comment (paulmillr) [inline README.md:41]: mobileconfig.mobileconfig.

File: profiles/mullvad-adblock-doh.mobileconfig (hunk 1 above)
Comment (sunknudsen) [reply profiles/mullvad-adblock-doh.mobileconfig:41]: Sure, will fix tomorrow morning.

File: README.md (hunk 2 above)
Comment (sunknudsen) [reply README.md:41]: Good catch, will fix tomorrow morning.

File: profiles/mullvad-adblock-doh.mobileconfig (hunk 1 above)
Comment (sunknudsen) [reply profiles/mullvad-adblock-doh.mobileconfig:41]: @paulmillr Fixed

Comment (paulmillr) [reply profiles/mullvad-adblock-doh.mobileconfig:41]: Nope. Payloadidentifier still differs from files in the repo.

Comment (sunknudsen) [reply profiles/mullvad-adblock-doh.mobileconfig:41]: Btw, you could be a little less blunt when people are contributing to your project.

Comment (sunknudsen) [reply profiles/mullvad-adblock-doh.mobileconfig]: @paulmillr Fixed

**Pull request merged**